gateway/response/response.go

```diff
@@ -105,7 +105,7 @@ func GetStatusCode(key string) int {
 	if status, ok := statusMap[key]; ok {
 		return status.code
 	}
-	return http.StatusOK
+	return http.StatusInternalServerError
```

Comment:
StatusInternalServerError is not always right. We might want to send StatusBadRequest also. 

This needs a discussion.